### ISSUE TITLE
Have c_rehash.in create links in the current directory to all other files in the directory list.

### ISSUE THREAD

**Issue by kfreidank** [openssl/openssl#15730]
"c_rehash" generates links in the same directory as the CRT and CRL files.
It would be convenient for c_rehash to generate the links in the current directory, linking to the files in each of the directories passed to it.
i.e. $ ls -l /etc/ssl/certs
lrwxr-xr-x  1 root  wheel  70 Jun 13 18:28 02265526.0 -> /usr/share/certs/trusted/Entrust_Root_Certification_Authority_-_G2.pem
lrwxr-xr-x  1 root  wheel  61 Jun 13 18:28 03179a64.0 -> /usr/share/certs/trusted/Staat_der_Nederlanden_EV_Root_CA.pem
... etc ...

**Comment by richsalz** [comment]
I think `c_rehash` should be be deprecated and `openssl rehash` should be the preferred way.  Perhaps the script could even just call the program :)

**Comment by ndusart** [comment]
`openssl rehash` seems to have the same limitation, or is there special option to create symlinks in current working directory ?

**Comment by t8m** [comment]
No, no such option there. But the comment means that the feature should be implemented in the openssl rehash and not in c_rehash.

**Comment by michael-o** [comment]
I support this request, in fact what should be supported is to provide the current list of input directories and the target with a switch. Most will pass: `"/etc/ssl/certs"` for the default verify location.

**Comment by nhorman** [comment]
@michael-o are you able to submit a PR for this?

**Comment by michael-o** [comment]
> @michael-o are you able to submit a PR for this?

Let me check how rehash currently works and what effort it would be to fix it. I'll get back to you in a couple of days.
